js/join.js

```diff
@@ -410,3 +424,81 @@ submit.addEventListener('click', async () => {
 copyBtn.addEventListener('click', () => {
   navigator.clipboard.writeText(`Real Dev Squad Verification Link: ${url}`);
 });
+
+function countWords(textArea) {
+  let m;
+  const regex = /\w+/g;
+  let count = 0;
+  while ((m = regex.exec(textArea)) !== null) {
+    if (m.index === regex.lastIndex) {
+      regex.lastIndex++;
+    }
+
+    m.forEach((match, groupIndex) => {
+      count++;
+    });
+  }
+  return count;
+}
+
+introduction.addEventListener('mouseleave', (event) => {
```

Comment:
Nit: why you used mouse leave event listener??

Reply:
The mouseleave event is triggered when the mouse pointer leaves that element and also user can also get idea from that . How much words are left. They don't need to be count manually .